Makefile

```diff
@@ -266,6 +266,11 @@ build-manifest: kustomize
 	./hack/build-manifest.sh "${OPTS}"
 .PHONY: build-manifest
 
+##@ Development env
+CLUSTER_PROVIDER ?= kind
+LOCAL_DEV_CLUSTER_VERSION ?= main
+KIND_WORKER_NODES ?=2
```

Comment:
this is a cool enhancement, we should pick it for other projects (e.g. PEAKS), cc @husky-parul